src/logservice/logminer/ob_log_miner_logger.cpp

```diff
@@ -79,8 +81,16 @@ int LogMinerLogger::log_progress(int64_t record_num, int64_t current_ts, int64_t
       nls_format, 0, time_buf, sizeof(time_buf), pos))) {
     LOG_WARN("datetime to string failed", K(current_ts), K(LOGMINER_TZ.get_tz_info()));
   } else {
-    fprintf(stdout, "\r%s %s %5.1lf%%, written records: %-20jd", time_buf, pb_buf,
-        progress, record_num);
+    int64_t current_time = ObTimeUtility::current_time();
+    int64_t speed = 0;
+    if (begin_ts_ == 0) {
+      begin_ts_ = current_time;
+    }
+    if (current_time - begin_ts_ > 0) {
+      speed = record_num * 1000 * 1000 / (current_time - begin_ts_);
```

Comment:
`1000 * 1000` is not easy to understand at a glance. you can use `_SEC_` or add some comments.